compiler/aarch64/codegen/ARM64OutOfLineCodeSection.cpp

```diff
@@ -39,5 +41,31 @@ void TR_ARM64OutOfLineCodeSection::assignRegisters(TR_RegisterKinds kindsToBeAss
 
 void TR_ARM64OutOfLineCodeSection::generateARM64OutOfLineCodeSectionDispatch()
    {
-   TR_UNIMPLEMENTED();
+   // Switch to cold helper instruction stream.
+   //
+   swapInstructionListsWithCompilation();
+
+   generateLabelInstruction(_cg, TR::InstOpCode::label, _callNode, _entryLabel);
+
+   TR::Register *resultReg = NULL;
+   if (_callNode->getOpCode().isCallIndirect())
+      resultReg = TR::TreeEvaluator::performCall(_callNode, true, _cg);
+   else
+      resultReg = TR::TreeEvaluator::performCall(_callNode, false, _cg);
+
+   if (_targetReg)
+      {
+      TR_ASSERT(resultReg, "assertion failure");
+      generateMovInstruction(_cg, _callNode, _targetReg, resultReg);
+      }
+   _cg->decReferenceCount(_callNode);
+
+   if (_restartLabel)
+      generateLabelInstruction(_cg, TR::InstOpCode::b, _callNode, _restartLabel);
+
+   generateLabelInstruction(_cg, TR::InstOpCode::label, _callNode, generateLabelSymbol(_cg));
```

Comment:
What is the purpose of this label instruction?   It doesn't have a "name" exposed outside of this instruction or any register dependencies hanging off of it, so its value is unclear to me.

Reply:
The line was copied from PPCOutOfLineCodeSection.cpp.
Removing.